app/decorators/atmosphere/migration_job_decorator.rb

```diff
@@ -0,0 +1,51 @@
+class Atmosphere::MigrationJobDecorator < Draper::Decorator
+  delegate_all
+
+  def appliance_type_name
+    if object.appliance_type
+      object.appliance_type.name
+    else
+      'unknown'
+    end
```

Comment:
Consider refactoring the if above into:
object.appliance_type ? object.appliance_type.name : 'unknown'